include/grpcpp/impl/codegen/server_context_impl.h

```diff
@@ -272,6 +279,36 @@ class ServerContext {
   /// Applications never need to call this method.
   grpc_call* c_call() { return call_; }
 
+  /// NOTE: This is an API for advanced users who need custom allocators.
+  /// Get and maybe mutate the allocator state associated with the current RPC.
+  /// Currently only applicable for callback unary RPC methods.
+  /// WARNING: This is experimental API and could be changed or removed.
+  ::grpc::experimental::RpcAllocatorState* GetRpcAllocatorState() {
+    return message_allocator_state_;
+  }
+
+  /// Get the default unary reactor for use in minimal reaction cases.
```

Comment:
We should make it clear that the user doesn't need to free or delete the returned reactor

Reply:
I added the words "library-owned" as well as a more detailed description to this comment.

Reply:
In reply to your other comment (which I somehow can't reply to through the UI), we're not using a factory precisely because this is supposed to avoid an allocation. I added a comment saying that they shouldn't call this more than once; although there's no enforcement mechanism now or even any negative impact, we may take advantage of this restriction in the future (e.g., to ease lazy initialization of the DefaultReactor).